docs/rss-feed-plan.md

```
@@ -0,0 +1,622 @@
+Extend `add-podcast` CLI command (or add `add-feed` alias):
+```bash
+doppler run -- python -m src.cli podcast add-feed https://example.com/rss
+```
+
+The existing `add-podcast` command should work for both since feed detection
+is automatic. An alias improves discoverability.
+
+### 11.2 Status command
+
+Update `status` to show article feed stats:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)